apps/evm/src/utilities/restService.ts

```diff
@@ -88,13 +88,8 @@ export async function restService<D>({
 
       try {
         data = await response.json();
-        const warning = response.headers.get('Warning');
-
-        if (warning) {
-          logError(warning);
-        }
-      } catch {
-        // Do nothing
+      } catch (err) {
+        logError(err);
```

Comment:
@therealemjy do you recall why we were ignoring errors here? I think at some point there was an error being spammed, but maybe we can have it being logged by the catch block again